app/statistics/page.tsx

```diff
@@ -0,0 +1,35 @@
+import { getBlockchains, getDailySummary, getTopListSummary } from 'services';
+
+import {
+  DEFAULT_STATISTIC_BREAK_DOWN_FILTER,
+  DEFAULT_STATISTIC_DAYS,
+} from 'constant';
+import StatisticsPageContent from './_components/StatisticsPageContent';
+import { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Statistics',
+};
+
+export default async function StatisticsPage() {
+  const blockchainsPromise = getBlockchains();
+  const dailySummaryPromise = getDailySummary({
+    days: DEFAULT_STATISTIC_DAYS,
+    breakDownBy: DEFAULT_STATISTIC_BREAK_DOWN_FILTER,
+  });
+  const topListSummaryPromise = getTopListSummary(DEFAULT_STATISTIC_DAYS);
+
+  const [blockchains, dailySummary, initialTopListSummary] = await Promise.all([
+    blockchainsPromise,
+    dailySummaryPromise,
+    topListSummaryPromise,
+  ]);
+
+  return (
+    <StatisticsPageContent
+      dailySummary={dailySummary}
```

Comment:
NIT: You can investigate using the app router caching mechanism to improve performance on the statistic page.

Reply:
It will be done in another PR to keep scope of this PR at this level as we spoke.